dotnet-desktop-guide/framework/winforms/controls/contextmenustrip-control-overview.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "ContextMenuStrip Control Overview"
+description: Learn about an overview of the ContextMenuStrip control, which replaces and adds functionality to the ContextMenu control.
```

Comment:
@mSarts Also, people don't learn about the overview of XYZ, they learn about XYZ be it an overview article or not. So you would use either something like `Overview of the ContextMenuStrip....` or `Learn about the ContextMenuStrip...`

Reply:
Addressed.